frontend/src/app/components/Header.js

```diff
@@ -152,6 +153,9 @@ export default class Header extends React.Component {
 }
 
 Header.propTypes = {
+  uninstallAddon: React.PropTypes.func.isRequired,
+  sendToGA: React.PropTypes.func.isRequired,
+  openWindow: React.PropTypes.func.isRequired,
   hasAddon: React.PropTypes.bool,
   forceHideSettings: React.PropTypes.bool
```

Comment:
[Added a commit to fix that](https://github.com/mozilla/testpilot/pull/1488/files#diff-014a13e8ad141df734228aa1890d34e8L38).

Missed writing a test for that, because I'm kind of confused [what this bit does](https://github.com/lmorchard/testpilot/blob/4f5bfc1f9da345359b4945d7a656b23df10b7bd9/frontend/src/app/components/Header.js#L22) and then forgot about it.